src/Qandidate/Common/Symfony/HttpKernel/EventListener/JsonRequestTransformerListener.php

```diff
@@ -20,6 +20,8 @@
  */
 class JsonRequestTransformerListener
 {
+    private $jsonContentTypes = ['json', 'application/json'];
```

Comment:
you should use the long array syntax, because of PHP 5.3 support (see Travis-CI logs)